apps/hash-external-services/docker-compose.ai.yml

```diff
@@ -19,6 +19,7 @@ services:
       HASH_TEMPORAL_SERVER_HOST: "http://temporal"
       HASH_TEMPORAL_SERVER_PORT: "${HASH_TEMPORAL_SERVER_PORT}"
       OPENAI_API_KEY: "${OPENAI_API_KEY}"
+      ANTHROPIC_API_KEY: "${ANTHROPIC_API_KEY}"
```

Comment:
Can you add this to the `README.md` for `hash-ai-worker-ts` please?

Reply:
Yes, have added in https://github.com/hashintel/hash/pull/4318/commits/78c33786919f4145861d0a2b9dc03ee98f3e7ee8